website/pages/docs/commands/namespace/apply.mdx

```diff
@@ -42,3 +42,21 @@ Create a namespace with a quota
 $ nomad namespace apply -description "Prod API servers" -quota prod api-prod
 Successfully applied namespace "api-prod"!
 ```
+
+Remove a quota from a namespace
+
+Quotas cannot be removed from a namespace once they have been applied, only replaced. To effectively remove a quota, create and apply an unlimited quota.
```

Comment:
is that true? 

```
$ nomad namespace apply -quota= test
Successfully applied namespace "test"!
~
$ nomad namespace inspect test
{
    "CreateIndex": 11,
    "Description": "",
    "ModifyIndex": 19,
    "Name": "test",
    "Quota": ""
}
```

Reply:
I know we don't like linking off to ENT but: https://github.com/hashicorp/nomad-enterprise/issues/414

Reply:
@tgross Thoughts? Would it be better to document the `quota= ` behavior for removal?

Reply:
Re-read my own issue... Ah, yeah that seems better. It's that `nomad quota delete` doesn't work quite as expected. So yeah let's document @cgbaker's solution.

Reply:
Done!